.ci/scripts/remove_simulator.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# The name here must match create_simulator.sh
+readonly DEVICE_NAME=Flutter-iPhone
+
+xcrun simctl shutdown "$DEVICE_NAME"
+xcrun simctl delete "$DEVICE_NAME"
```

Comment:
I think you can delete without shutting down?

Reply:
Actually this matches what we do in the framework:
https://github.com/flutter/flutter/blob/dfab19c1fe9d2e2f359494700c80633f0404cf31/dev/devicelab/lib/framework/ios.dart#L115-L131